django/applications/catmaid/control/graph2.py

```diff
@@ -356,7 +354,7 @@ def split_by_both(skeleton_id, digraph, locations, bandwidth, cs, connectors, in
             chunk[parent][child]['weight'] = norm(subtract(locations[child], locations[parent]))
 
         # Check if need to expand at all
-        blob = tuple(c for c in cs if c[0] in chunk) # type: Optional[Tuple]
+        blob:Optional[Tuple] = tuple(c for c in cs if c[0] in chunk)
```

Comment:
Making it optional is to handle the anchors.get() use much later in this block, although that really should be refactored into being a distinct variable. I'll do that so this can just be a Tuple.

Reply:
Ah, didn't look that far down - good catch. I agree that changing the name of the latter variable is a good way to go.